build/release/Makefile

```diff
@@ -19,19 +19,33 @@ include ../makelib/common.mk
 # ====================================================================================
 # Options
 
+# set the shell to bash in case some environments use sh
+SHELL := /bin/bash
+
 CHANNEL ?= master
 ifeq ($(filter master alpha beta stable,$(CHANNEL)),)
 $(error invalid channel $(CHANNEL))
 endif
 
+DOCS_VERSION := $(shell echo $(BRANCH_NAME) | sed -E "s/^release\-([0-9]+)\.([0-9]+)$$/v\1.\2/g")
+DOCS_DIR ?= $(ROOT_DIR)/Documentation
+DOCS_WORK_DIR := $(WORK_DIR)/rook.github.io
+DOCS_VERSION_DIR= $(DOCS_WORK_DIR)/docs/rook/$(DOCS_VERSION)
+
+ifdef GIT_API_TOKEN
+DOCS_GIT_REPO := https://$(GIT_API_TOKEN)@github.com/ilovett/ilovett.github.io.git
+else
+DOCS_GIT_REPO := git@github.com:ilovett/ilovett.github.io.git
+endif
+
 ifeq ($(origin BRANCH_NAME), undefined)
 BRANCH_NAME := $(shell git branch | grep \* | cut -d ' ' -f2)
 endif
 
 PLATFORMS ?= $(ALL_PLATFORMS)
 
 ifneq ($(filter master release-%,$(BRANCH_NAME)),)
```

Comment:
this filter needs to be more specific `/^release\-[0-9]\.[0-9]+$/` to avoid competing HEAD of different branches matching the same major/minor overwriting each other.  the branch names should never include patch version or anything else tailing on the end.  my Makefile-fu is not great enough to do this a good way.